.github/workflows/cpp-python-tests.yml

```diff
@@ -4,105 +4,129 @@ name: cpp-python-tests
 on:
   pull_request:
     branches: [main]
+  push:
+    branches: [main]
+  workflow_dispatch:
+    inputs:
+      pytest_stdout:
+        description: "Show test STDOUT in log for Pytest tests"
+        required: false
+        type: boolean
+        default: false
```

Comment:
Allows to manually start the workflow with an option to show all the details on pytest tests